napari/_vispy/canvas.py

```diff
@@ -300,7 +301,10 @@ def delete(self) -> None:
 
     def _on_interactive(self) -> None:
         """Link interactive attributes of view and viewer."""
-        self.view.interactive = self.viewer.camera.interactive
+        # Is this should be changed or renamed?
```

Comment:
Do you mean should `self.view.interactive` be renamed? It looks like `self.view` is a vispy `ViewBox`, so I don't think we can rename that.

Maybe we could rename `_on_interactive` to `_on_mouse_pan_or_zoom`?

Don't feel strongly though.

Reply:
I do not know. I do minimum work to rip of using deprecated event.

Reply:
Well, I'm fine with the minimum here :) It's understandable enough for me.